R/zchunk_LA1011.en_bal_adj.R

```diff
@@ -1,88 +1,214 @@
+      inner_join(EIA_years, by = "year") %>%
+      mutate(value_diff = value_TOT - value) %>%
+      select(Country, year, value_diff) %>%
+      mutate(value = value_diff * CONV_KBBL_BBL * CONV_BBL_TONNE_DISTILLATE *
+               CONV_TONNE_GJ_RFO * CONV_GJ_EJ / CONV_DAYS_YEAR) %>%
+      select(Country, year, value) -> L1011.in_EJ_ctry_intlship_distillate_Yh
+
+    L1011.in_EJ_ctry_intlship_distillate_Yh %>%
+      left_join(L1011.in_EJ_ctry_intlship_RFO_Yh %>%
+                  rename(value_RFO = value), by = c("year", "Country")) %>%
+      mutate(value = value + value_RFO) %>%
+      select(Country, year, value) -> L1011.in_EJ_ctry_intlship_TOT_Yh
+
+    # Several countries blink in and out of the time series. Only changing Russia because it's the only one that is a really large amount of fuel
+    L1011.in_EJ_ctry_intlship_TOT_Yh %>%
+      mutate(value = if_else(is.na(value), 0, value)) -> L1011.in_EJ_ctry_intlship_TOT_Yh
+
+    L1011.in_EJ_ctry_intlship_TOT_Yh %>%
+      filter(Country == "Russia" & year %in% 2005 ) %>%
+      select(value) -> Russia_2005
+
+    L1011.in_EJ_ctry_intlship_TOT_Yh %>%
+      mutate(value = replace(value, Country == "Russia" & year %in% c(2006, 2007, 2008, 2009, 2010),
+                             Russia_2005$value)) -> L1011.in_EJ_ctry_intlship_TOT_Yh
+
+    # Russia's energy use prior to 1991 is assigned to Former Soviet Union; need to re-map this to Russia
+    # for country-level information being written out
+    L1011.in_EJ_ctry_intlship_TOT_Yh %>%
+      filter(!(Country == "Russia" & year %in% c(1986, 1987, 1988, 1989, 1990, 1991))) %>%
+      mutate(Country = replace(Country, Country == "Former U.S.S.R." & year %in% c(1986, 1987, 1988, 1989, 1990, 1991),
+                               "Russia")) -> L1011.in_EJ_ctry_intlship_TOT_Yh
+
+    # Match in the countries and aggregate by region
+    L1011.in_EJ_ctry_intlship_TOT_Yh %>%
+      left_join(EIA_ctry %>%
+                  rename(Country = EIA_ctry), by = "Country") %>%
+      left_join(iso_GCAM_regID, by = "iso") %>%
+      filter(!(is.na(GCAM_region_ID))) %>%
+      select(Country, year, value, iso, GCAM_region_ID) -> L1011.in_EJ_ctry_intlship_TOT_Yh
+
+    L1011.in_EJ_ctry_intlship_TOT_Yh %>%
+      select(GCAM_region_ID, year, value) %>%
+      group_by(GCAM_region_ID, year) %>%
+      summarize(value = sum(value)) -> L1011.in_EJ_R_intlship_Yh
+
+    # Replace the data in the IEA energy balances table
+    L101.en_bal_EJ_R_Si_Fi_Yh_full %>%
+      left_join(L1011.in_EJ_R_intlship_Yh %>%
+                  mutate(sector = c("in_trn_international ship")) %>%
+                  mutate(fuel = c("refined liquids")),
+                by = c("sector", "fuel", "year", "GCAM_region_ID")) %>%
+      mutate(value.x = if_else(is.na(value.y), value.x, value.y)) %>%
+      rename(value = value.x) %>%
+      select(GCAM_region_ID, sector, fuel, year, value) -> L1011.en_bal_EJ_R_Si_Fi_Yh
```

Comment:
`select(GCAM_region_ID, sector, fuel, year, value = value.x)` and get rid of line 129